components/03-organisms/site/site-header/site-header.twig

```diff
@@ -1,5 +1,11 @@
 {% set header__base_class = 'header' %}
 
+{% if directory %}
+  {% set header__logo_src = icon_url|default('/' ~ directory ~ '/images/logo.png') %}
+{% else %}
+  {% set header__logo_src = 'images/logo.png' %}
+{% endif %}
```

Comment:
Same thing as above, this could be a one-liner ternary:
`{% set header__logo_src = directory ? '/' ~ directory ~ '/images/logo.png' : 'images/logo.png' %}`

Reply:
Done